framework/source/class/qx/Theme.js

```diff
@@ -57,7 +57,8 @@ qx.Bootstrap.define("qx.Theme",
      *   fonts : {},
      *   widgets : {},
      *   appearances : {},
-     *   meta : {}
+     *   meta : {},
+     *   boot : {}
```

Comment:
It appears that this example is wrong, and doing as the example suggests would crash the program. The `boot` key is expected to point to a function, not a map as shown here.

Reply:
Any recommendations on how to best show? So far i have this: "boot : {function(){}}"